packages/sankey/src/SankeyLabels.tsx

```diff
@@ -109,7 +118,7 @@ export const SankeyLabels = <N extends DefaultNode, L extends DefaultLink>({
                         }}
                     >
                         {label.label}
```

Comment:
I think the component should decide what to do with the label, not have it as a children, for example, if you have long labels, you could want to split the text.

Reply:
Don't you have issues with the type of `children`?